distribution/helm/vector/values.yaml

```diff
@@ -55,16 +55,16 @@ securityContext: {}
 # Extra env vars to pass to the `vector` container.
 env: []
 
-# Various tweakables for the `Pod`s managed by `DaemonSet`.
+# Various tweaks for the `Pod`s managed by `DaemonSet`.
```

Comment:
The original might have been right—these are values that are able to be tweaked. @MOZGIII?

Reply:
I saw this, it looks like the semantics for the two would be identical in this case. I intended to use "tweakables" there, i.e. it's not an error per se - but it may be odd or unconventional wording, while simple and relatively more common "tweaks" seem to deliver the same idea.
So, it is a slight change in meaning, but I don't mind if this form is more appealing to native speakers.

Reply:
👍 Swapped it back to 'tweakables'. Jumped out at me initially, but makes sense now.